pkgs/development/python-modules/mat2/default.nix

```diff
@@ -49,16 +49,6 @@ buildPythonPackage rec {
     ./executable-name.patch
     # hardcode path to mat2 executable
     ./tests.patch
-    # fix gobject-introspection typelib path for Nautilus extension
-    (substituteAll {
-      src = ./fix_poppler.patch;
```

Comment:
the patch is deleted, but the file remains... Is this wanted?
Did you test the nautilus-plugin? because below you remove the nautilus-extension as well.

Reply:
nevermind the extension got removed. But please remove the patch-file `./fix_poppler.patch`.